File: example/javalib/testing/7-test-quick/foo/test/src/FooTest1.java
package foo;

import org.junit.Test;
import static org.junit.Assert.*;

public class FooTest1 {
    @Test
    public void test1() {
        String name = "Aether";
        String greeted = Foo.greet(name);
        assertEquals("Hello, " + name + "!", greeted);
    }
}

File: example/kotlinlib/testing/7-test-quick/foo/test/src/FooTest1.kt
package foo

import org.junit.jupiter.api.Test
import org.junit.jupiter.api.Assertions.*

class FooTest1 {
    @Test
    fun test1() {
        val name = "Aether"
        val greeted = Foo.greet(name)
        assertEquals("Hello, $name!", greeted)
    }
}